### PR TITLE
create modal to confirm job post created

### PR DESCRIPTION
**Pull request by CharlesMing02**
## Documentation
- added modal to confirm job posting
- imported dm sans font
- added typography global styling

## Related PRs
Everyone should be able to use dm sans after this. Reminder for everyone to run yarn install.

## Screenshots
IMG_E5EF1C872CD9-1

CC: @alliu879 @surajr10 @cecilia-gao3

### CODE REVIEW

File: src/screens/Drafting/Draft.tsx
Comment (alliu879) [inline src/screens/Drafting/Draft.tsx]: remove comment

File: src/screens/Drafting/Draft.tsx
Comment (alliu879) [inline src/screens/Drafting/Draft.tsx:252]: just checking — what is this `goBack()` used for?

File: src/components/StyledButton/StyledButton.tsx
Comment (alliu879) [inline src/components/StyledButton/StyledButton.tsx]: is this the same button that annie added to her auth styling PR? if not this may need to be renamed

**Review by alliu879 — APPROVED** [review]
looks really good!! approving with a few nits :)

File: src/screens/Drafting/Draft.tsx
Comment (CharlesMing02) [reply src/screens/Drafting/Draft.tsx]: fixed good catch

File: src/screens/Drafting/Draft.tsx
Comment (CharlesMing02) [reply src/screens/Drafting/Draft.tsx:252]: it gets rid of the previous job post screen before navigating to the new screen. I thought it would feel more intuitive and like the form is fully submitted. Ultimately a UX decision so I'll do what you and Ceci think is best

File: src/components/StyledButton/StyledButton.tsx
Comment (CharlesMing02) [reply src/components/StyledButton/StyledButton.tsx]: Yeah, I can pull her new PR from main and make sure there aren't duplicates or conflicts before merging

**Pull request merged**
